File: 1791-richest-customer-wealth/richest-customer-wealth.py
class Solution(object):
    def maximumWealth(self, accounts):
        """
        :type accounts: List[List[int]]
        :rtype: int
        """
        result = 0
        for i in range(0,len(accounts)):
            dummy =sum(accounts[i])
            result = max(result , dummy)
        return(result)

File: 1791-richest-customer-wealth/richest-customer-wealth.cs
public class Solution {
    public int MaximumWealth(int[][] accounts) {
        int result = 0;
			foreach (var i in accounts)
			{
				if (i.Sum() > result )
				{
					result = i.Sum();
				}
			}

			return (result);

        
    }
}